python/oneflow/framework/tensor_str.py

```diff
@@ -34,14 +34,12 @@ class __PrinterOptions(object):
 PRINT_OPTS = __PrinterOptions()
 
 
-def _convert_to_local_tensor(self):
-    # consistent to local
-    if self.is_consistent:
-        placement = flow.placement("cpu", {0: [0]})
-        sbp = flow.sbp.broadcast
-        # TODO: delete `to("cuda")` after supporting cpu data broadcast
-        self = self.to("cuda").to_consistent(placement, sbp).to_local()
-    return self
+def _try_convert_to_local_tensor(tensor):
+    return (
+        tensor
+        if tensor.is_local
+        else tensor.to_consistent(sbp=flow.sbp.broadcast).to_local()
+    )
```

Comment:
```suggestion
    if tensor.is_consistent:
        tensor = tensor.to_consistent(sbp=flow.sbp.broadcast).to_local()
    return tensor
```
这样写好一点，函数内部就不要用这种语法了